src/display/api.js

```diff
@@ -26,6 +26,7 @@ import {
   RenderingCancelledException, StatTimer
 } from './dom_utils';
 import { FontFaceObject, FontLoader } from './font_loader';
```

Comment:
It looks like the commit message of this commit is copy-pasted from the first one ;-)

Reply:
I've tried to trim the commit messages a bit.